src/main/frontend/vue-app/components/SignInScreen.vue

```diff
@@ -175,12 +175,28 @@ html {
               }
             }
           }
-          .uiIconUser {
-            color: #999;
+          .guestName input {
+            box-shadow: none;
+            font-size: 15px;
+          }
+          .iconUser {
             position: absolute;
-            top: 15%;
-            left: 4%;
+            top: 4%;
+            left: 1%;
+            height: 30px;
+            width: 30px;
             font-size: 20px;
+            background: #ffffff url(/eXoSkin/skin/images/themes/default/username.png) no-repeat left -3px;
+            background-image: url(/eXoSkin/skin/images/themes/default/username.png);
+            background-position-x: left;
+            background-position-y: -3px;
+            background-size: initial;
+            background-repeat-x: no-repeat;
+            background-repeat-y: no-repeat;
```

Comment:
I think there are duplicated CSS rules here, could you remove unneeded ones ?

Reply:
We have cleaned CSS